<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>837</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>581</width>
      <height>541</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Mesh Showing Frame&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <zorder>label</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>10</y>
      <width>201</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Info Pannel</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>#Num Of Points</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="NumOfPointsText">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="NumOfLinesText">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>#Num Of Lines</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>#Num Of Meshes</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="NumOfCellsText">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>201</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>Control Pannel</string>
    </property>
    <widget class="QRadioButton" name="POP">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adjacent Points (Input Point ID)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MOP">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adjacent Meshes (Input Point ID)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="MOM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adjacent Meshes (Input Mesh ID)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="NOM">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>181</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mesh Normal (Input Mesh ID)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputID">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>181</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonDraw">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Draw!</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="AP3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Area Paint(3)</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="AP2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Area Paint(2)</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>837</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuWhat">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionMesh"/>
    <addaction name="actionColor"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuWhat"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionMesh">
   <property name="text">
    <string>Open Mesh</string>
   </property>
  </action>
  <action name="actionColor">
   <property name="text">
    <string>Open Color</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
